charts/mattermost-calls-offloader/templates/hpa.yaml

```diff
@@ -0,0 +1,28 @@
+{{- if .Values.autoscaling.enabled }}
+apiVersion: autoscaling/v2beta1
```

Comment:
I think there is v2 stable
https://github.com/kubernetes/api/tree/master/autoscaling

Reply:
Nice catch @angeloskyratzakos thanks. Fixed ;)